openlibrary/templates/books/edit/edition.html

```diff
@@ -304,7 +267,7 @@
 
             <legend>$_('Languages')</legend>
 
-            <div class="formElement" id="languages">
+            <div class="formElement language_multi_input_autocomplete">
```

Comment:
Can I ask that we use BEM notation for this class for consistency with newer classes ? http://getbem.com/

Reply:
This is my first time reading about BEM. I get the general idea but I'm not confident in how it applies in this situation where there is no styling and the class is being used as a hack until we have proper components.

Anyway, I changed this to `multi-input-autocomplete--language`. The interpretation is that `multi-input-autocomplete` is the block since that's the main thing we're operating on. Then `--language` for a modifier since this class is ultimately being used to modify the block to work for language.

How does that sound?